docs/automate.md

```diff
@@ -26,9 +26,13 @@ Run as a one-liner. NOTE: replace with yourorg/yourrepo (repo level) or just you
 curl -s https://raw.githubusercontent.com/actions/runner/automate/scripts/create-latest-svc.sh | bash -s yourorg/yourrepo
 ```
 
+### Why can't I use a container?
```

Comment:
why do we need this?

Reply:
I tested out the `create-svc.sh` and `remove-svc.sh` on a VM using these docs.

I thought we could further discourage trying it on a container by offering an explanation why the runner can't be installed as a service on docker.